aws-xray-recorder-sdk-core/src/test/java/com/amazonaws/xray/emitters/UDPEmitterTest.java

```diff
@@ -1,21 +1,44 @@
 package com.amazonaws.xray.emitters;
 
 import com.amazonaws.xray.config.DaemonConfiguration;
-import org.junit.Assert;
+import com.amazonaws.xray.entities.DummySegment;
 import org.junit.Test;
 
 import java.net.SocketException;
 
+import static com.amazonaws.xray.AWSXRay.getGlobalRecorder;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
 public class UDPEmitterTest {
 
     @Test
     public void testCustomAddress() throws SocketException {
         String address = "123.4.5.6:1234";
-        DaemonConfiguration config = new DaemonConfiguration();
-        config.setDaemonAddress(address);
+        DaemonConfiguration config = getDaemonConfiguration(address);
 
         UDPEmitter emitter = new UDPEmitter(config);
 
-        Assert.assertEquals(address, emitter.getUDPAddress());
+        assertEquals(address, emitter.getUDPAddress());
+    }
+
+
+    @Test
+    public void sendingSegmentShouldNotThrowExceptions() throws SocketException {
+        DaemonConfiguration config = getDaemonConfiguration("host:1234");
+        UDPEmitter emitter = new UDPEmitter(config);
+
+        try {
+            final boolean success = emitter.sendSegment(new DummySegment(getGlobalRecorder()));
+            assertEquals(false, success);
+        } catch (Exception e) {
+            fail();
```

Comment:
An exception will cause the test to fail anyways so we don't need to catch, can you remove the try clause?